java/java.completion/src/org/netbeans/modules/java/completion/JavaCompletionTask.java

```diff
@@ -22,6 +22,7 @@
 import com.sun.source.tree.*;
 import com.sun.source.tree.Tree.Kind;
 import com.sun.source.util.*;
+import com.sun.tools.javac.tree.JCTree.JCExpression;
```

Comment:
Please no internal types inside language features (besides the language infrastructure, like java.source{.base}, etc.).